docs/supported-technologies.asciidoc

```diff
@@ -39,15 +39,26 @@ If you are interested in accurately measuring the duration of load event, the in
 
 
 [float]
-[[platforms]]
-=== Platforms
+[[user-interactions]]
+=== User Interactions
 
-The following platforms are supported:
+The RUM agent automatically instruments click event listeners that are
+registered by your application. The click events are captured as `user-interaction`
+transactions. However, to avoid sending too many `user-interaction` transactions
```

Comment:
Instead of saying it this way, We can explain like. Not all click events would be interesting and the agent would only capture the listeners as user-interactions when there is a network activity.

Reply:
I've rephrased it a bit.